Sources/client-sdk-swift/config/TopicConfigurations.swift

```diff
@@ -0,0 +1,7 @@
+let defaultLoggerFactory = DefaultMomentoLoggerFactory()
+
+public class Default: TopicClientConfiguration {
+    static func latest() -> TopicClientConfiguration {
+        return TopicClientConfiguration(loggerFactory: defaultLoggerFactory, transportStrategy: StaticTransportStrategy(grpcConfig: StaticGrpcConfiguration(deadlineMillis: 15)))
```

Comment:
This seems awfully low for the deadline value. I imagine we want something closer to 15 seconds for a mobile platform (going by the corresponding .NET value for mobile topic deadline).